website/sidebars.js

```diff
@@ -302,12 +302,12 @@ module.exports = {
                     },
                 },
                 "troubleshooting/access",
-                "troubleshooting/csrf",
-                "troubleshooting/emails",
                 "troubleshooting/login",
                 "troubleshooting/image_upload",
                 "troubleshooting/missing_permission",
                 "troubleshooting/missing_admin_group",
+                "troubleshooting/csrf",
```

Comment:
all looks good - I just wonder why the positions of these have to change.

Reply:
![image](https://github.com/goauthentik/authentik/assets/1932513/98608202-1539-47b5-94a9-ce147e75fa07)
with the new titles the new order makes more sense imo